langchain/document_loaders/notiondb.py

```diff
@@ -19,14 +19,20 @@ class NotionDBLoader(BaseLoader):
     Args:
         integration_token (str): Notion integration token.
         database_id (str): Notion database id.
+        request_timeout_sec (int): Timeout for Notion requests in seconds.
     """
 
-    def __init__(self, integration_token: str, database_id: str) -> None:
+    def __init__(self, integration_token: str, database_id: str, request_timeout_sec: int = None) -> None:
         """Initialize with parameters."""
         if not integration_token:
             raise ValueError("integration_token must be provided")
         if not database_id:
             raise ValueError("database_id must be provided")
+        if request_timeout_sec > 30:
+            raise ValueError("request timeout cannot exceed 30s")
+
+        if not request_timeout_sec:
+            request_timeout_sec = 10
```

Comment:
Why it is 10, usually timeouts are set to 10. And if you want to default value to 10 then set it on param.

Reply:
got it, i set the default value in the function argument instead.  it's 10s here bc that was the hardcoded value before